pgamit/pyETM.py

```diff
@@ -2701,22 +2788,27 @@ def get_xyz_s(self, year, doy, jmp=None, sigma_h=SIGMA_FLOOR_H, sigma_v=SIGMA_FL
                 source += '. fast moving station, bumping up sigmas'
 
         # apply floor sigmas
-        sig = np.sqrt(np.square(sig) + np.square(np.array([[sigma_h], [sigma_h], [sigma_v]])))
+        sig = np.sqrt(np.square(sig) + np.square(np.array(
+            [[sigma_h], [sigma_h], [sigma_v]])))
```

Comment:
@pdsmith90 a bit more readable if each term being added is kept grouped together with the operand:

```python
        sig = np.sqrt(np.square(sig) + 
                      np.square(np.array([[sigma_h], [sigma_h], [sigma_v]])))
```